android/app/build.gradle

```diff
@@ -250,27 +250,34 @@ android {
     }
 
     packagingOptions {
-        pickFirst 'lib/arm64-v8a/liblog.so'    
-        pickFirst 'lib/armeabi-v7a/liblog.so'  
+        pickFirst 'lib/arm64-v8a/liblog.so'
+        pickFirst 'lib/armeabi-v7a/liblog.so'
         pickFirst 'lib/x86/liblog.so'
         pickFirst 'lib/x86_64/liblog.so'
         resources {
             excludes += "META-INF/*"
             excludes += "META-INF/spring/aot.factories"
-         }
+        }
     }
 }
 
 dependencies {
-    implementation("io.mosip:inji-openid4vp:0.2.0-SNAPSHOT")
+    implementation("io.mosip:inji-openid4vp:0.3.0-SNAPSHOT"){
+        changing = true
+        exclude group: 'org.bouncycastle', module: 'bcpkix-jdk15on'
+        exclude group: 'org.bouncycastle', module: 'bcpkix-jdk18on'
+        exclude group: 'io.mosip', module: 'vcverifier-jar'
+        exclude group: 'com.google.crypto.tink', module: 'tink'
+        exclude group: 'com.augustcellars.cose', module:'cose-java'
```

Comment:
This libary is [BSD-3-Clause license] was the license compatiblity and attribution requirement were checked ?